src/main/java/com/exasol/adapter/dynamodb/documentfetcher/dynamodb/DynamodbQueryDocumentFetcherFactory.java

```diff
@@ -0,0 +1,87 @@
+package com.exasol.adapter.dynamodb.documentfetcher.dynamodb;
+
+import java.util.List;
+
+import com.amazonaws.services.dynamodbv2.model.QueryRequest;
+import com.exasol.adapter.dynamodb.documentfetcher.DocumentFetcher;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbNodeVisitor;
+import com.exasol.adapter.dynamodb.dynamodbmetadata.DynamodbIndex;
+import com.exasol.adapter.dynamodb.dynamodbmetadata.DynamodbPrimaryIndex;
+import com.exasol.adapter.dynamodb.dynamodbmetadata.DynamodbSecondaryIndex;
+import com.exasol.adapter.dynamodb.dynamodbmetadata.DynamodbTableMetadata;
+import com.exasol.adapter.dynamodb.remotetablequery.RemoteTableQuery;
+import com.exasol.adapter.dynamodb.remotetablequery.normalizer.DnfNormalizer;
+import com.exasol.adapter.dynamodb.remotetablequery.normalizer.DnfOr;
+
+/**
+ * This factory builds {@link DynamodbQueryDocumentFetcher}s for a given query. If the query can't be solved by a single
+ * Query operation but using a few, this factory generates multiple operations.
```

Comment:
```suggestion
 * {@code Query} operation but using a few, this factory generates multiple operations.
```

Reply:
If you have link to what is query operation, you can also add it instead of code.